pjmedia/src/pjmedia-codec/openh264.cpp

```diff
@@ -1056,7 +1057,18 @@ static pj_status_t oh264_codec_decode(pjmedia_vid_codec *codec,
 	start = oh264_data->dec_buf + buf_pos;
 
 	/* Decode */
-	oh264_data->dec->DecodeFrame2( start, frm_size, pData, &sDstBufInfo);
+	ret = oh264_data->dec->DecodeFrame2( start, frm_size, pData,
+					     &sDstBufInfo);
+
+	if (ret != dsErrorFree && !kf_requested) {
```

Comment:
why is !kf_requested check necessary here since the value must be FALSE? for future use?

Reply:
just to avoid multiple events per frame decoding, note that this decode is inside a loop.